src/eval.ml

```diff
@@ -166,6 +176,126 @@ and eq_modulo : term -> term -> bool = fun a b ->
   let res = try eq_modulo [(a,b)]; true with Exit -> false in
   if !log_enabled then log_eqmd (r_or_g res "%a == %a") pp a pp b; res
 
+(** [specialize p m]  specializes the matrix [m] when matching against pattern
```

Comment:
The following code should be in dtree.ml.

Reply:
Done in 3bce2a1.